### DISCUSSION TITLE
Copilot version confusion

### DISCUSSION THREAD

**Discussion by rameshsanth** [zbirenbaum/copilot.lua#303] [General]
Latest [copilot.vim release](https://github.com/github/copilot.vim/commits/release/) is 1.38.0
I am using latest copilot.lua. But version (<cmd> lua Copilot version) from copilot.lua plugin shows its using copilot.vim 1.13.0 which is 7 months old.
Am I really using latest copilot or a 7 month old one ?

![Screenshot 2024-07-25 at 8 52 42 AM](https://github.com/user-attachments/assets/2cc972bc-b885-4c99-a5c4-d837a5a3021d)

**Comment by bulletmark** [comment]
I just checked my versions in LazyVim and still says `copilot.vim` is at 1.13.0 but [current version available](https://github.com/github/copilot.vim/tags) is 1.41.0. There surely is an easy explanation for this?

**Comment by AntoineGS** [comment]
It has been updated a few times but the version number was just never changed :P
